### ISSUE TITLE
BackwardErrmeasure bad name 

### ISSUE THREAD

**Issue by jarlebring** [nep-pack/NonlinearEigenproblems.jl#212]
The `BackwardErrmeasure` which we define by the formula for SPMF-NEPs:
![image](https://user-images.githubusercontent.com/11163595/64475880-a37ac400-d188-11e9-877a-6c1b53917850.png)
is actually a scaled backward error. If it was actually the backward error it wouldn't be the best thing to use as stopping criteria. I would suggest we rename this to `SPMFErrmeasure` or `StandardErrmeasure`. I'm slightly in favor of `StandardErrmeasure`.

**Comment by eringh** [comment]
I would say `StandardSPMFErrmeasure`? The measure works for `AbstractSPMF`, right?

However, it currently accepts any `NEP`. A minimum working example (error) is:
```
function my_Mder(s,der)
    A0=ones(Float64,3,3)-I; A0[1,1]=-1;
    A1=ones(Float64,3,3)*3; A1[2,3]=0;
    if (der==0)
       return A0+A1*s;
    elseif (der==1)
       return A1;
    else
       return zero(A0);
    end
end
nep=Mder_NEP(3,my_Mder);
BackwardErrmeasure(nep)
```
which results in:
```
ERROR: MethodError: no method matching get_Av(::Mder_Mlincomb_NEP)
```

(I can fix the issue, which is a separate thing. However, I think it justifies that SPMF should be in the name)

**Comment by jarlebring** [comment]
I agree with both things. `StandardSPMFErrmeasure` is more precise. That specific function should only be defined for `AbstractSPMF`.
